app/Http/Middleware/SetLanguage.php

```diff
@@ -22,10 +22,11 @@ class SetLanguage
      * @param Closure $next
      * @return mixed
      */
-    public function handle($request, Closure $next)
+    public function handle(Request  $request, Closure $next)
     {
         $routeLocaleValue = $request->route('locale');
 
+        //@todo this needs to be testes if strict will affect
         if (!empty($routeLocaleValue) && in_array($routeLocaleValue, self::ACCEPTED_LANGUAGES)) {
```

Comment:
folosim lista de coduri din config

```suggestion
        if (!empty($routeLocaleValue) && in_array($routeLocaleValue, config('translatable.locales'))) {
```